tests/integration/tasks/test_task_utils.py

```diff
@@ -0,0 +1,20 @@
+from viadot.task_utils import custom_mail_state_handler
+from prefect.tasks.secrets import PrefectSecret
+from prefect.engine.state import Failed, Success
+import prefect
```

Comment:
Is this import needed here? Can't you use `Failed` line 20 instead of `prefect.engine.state.Failed`? You are using `Failed` in line 13 so maybe it can be used in the same way in line 20.